przydatne_linki.md

```diff
@@ -15,8 +15,12 @@ Terminy egzaminów: https://zapisy.ii.uni.wroc.pl/session/
 
 Mail: https://outlook.office.com/
 
-Program studiów (Informatyka): https://ii.uni.wroc.pl/media/uploads/Program_studiow_Inf_LI(2)_WECibSo.pdf
+Program studiów (Informatyka) (2019-2021): https://ii.uni.wroc.pl/media/uploads/Program_studiow_Inf_LI(2)_WECibSo.pdf
 
-Program studiów (ISIM): https://ii.uni.wroc.pl/media/uploads/2022/11/29/2022_isim_program.pdf
+Program studiów (Informatyka) (od 2022): https://ii.uni.wroc.pl/media/uploads/2022/09/22/informatyka-i-skroconydocx.pdf
+
+Program studiów (ISIM) (2019-2021): https://www.wmi.uni.wroc.pl/sites/default/files/upload_attach/2019isimprogram.pdf
```

Comment:
To ten sam dokument co poprzedni, ale w domenie wmi zamiast ii. Zrobiłem tak żeby było spójne z linkiem od 2022